Contributing Guidelines (#1)
TheAlgorithms/Nim: README.md

<div align="center">
<!-- Title: -->
  <a href="https://github.com/TheAlgorithms/">
    <img src="https://raw.githubusercontent.com/TheAlgorithms/website/1cd824df116b27029f17c2d1b42d81731f28a920/public/logo.svg" height="100">
  </a>
  <h1><a href="https://github.com/TheAlgorithms/">The Algorithms</a> - Nim</h1>
<!-- Labels: -->
  <!-- First row: -->
  <a href="https://gitpod.io/#https://github.com/TheAlgorithms/Nim">
    <img src="https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod&style=flat-square" height="20" alt="Gitpod Ready-to-Code">
  </a>
  <a href="https://github.com/TheAlgorithms/Nim/blob/master/CONTRIBUTING.md">
    <img src="https://img.shields.io/static/v1.svg?label=Contributions&message=Welcome&color=0059b3&style=flat-square" height="20" alt="Contributions Welcome">
  </a>
  <img src="https://img.shields.io/github/repo-size/TheAlgorithms/Nim.svg?label=Repo%20size&style=flat-square" height="20">

>   <a href="https://discord.gg/c7MnfGFGa6">

Reviewer: ```suggestion
  <a href="https://the-algorithms.com/discord">
```

    <img src="https://img.shields.io/discord/808045925556682782.svg?logo=discord&colorB=7289DA&style=flat-square" height="20" alt="Discord chat">
  </a>
  <a href="https://gitter.im/TheAlgorithms">
    <img src="https://img.shields.io/badge/Chat-Gitter-ff69b4.svg?label=Chat&logo=gitter&style=flat-square" height="20" alt="Gitter chat">
  </a>
  <!-- Second row: -->
  <br>

>   <a href="https://github.com/TheAlgorithms/Nim/actions">
>     <img src="https://img.shields.io/github/actions/workflow/status/TheAlgorithms/Nim/build.yml?branch=master&label=CI&logo=github&style=flat-square" height="20" alt="GitHub Workflow Status">
>   </a>
>   <a href="https://github.com/pre-commit/pre-commit">
>     <img src="https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white&style=flat-square" height="20" alt="pre-commit">
>   </a>
>   <a href="https://github.com/psf/black">
>     <img src="https://img.shields.io/static/v1?label=code%20style&message=black&color=black&style=flat-square" height="20" alt="code style: black">
>   </a>

Reviewer: These can be removed as we don't have CI yet, nor do we use pre-commit or the Python formatter.
```suggestion
```

<!-- Short description: -->
  <h3>All algorithms implemented in Nim - for education</h3>
</div>

Implementations are for learning purposes only. They may be less efficient than the implementations in the [Nim standard library](https://nim-lang.org/docs/lib.html) or [Nimble packages](https://nimble.directory/). Use them at your discretion.

## Getting Started

Read through our [Contribution Guidelines](CONTRIBUTING.md) before you contribute.

## Community Channels

> We are on [Discord](https://discord.gg/c7MnfGFGa6) and [Gitter](https://gitter.im/TheAlgorithms)! Community channels are a great way for you to ask questions and get help. Please join us!

Reviewer: ```suggestion
We are on [Discord](https://the-algorithms.com/discord) and [Gitter](https://gitter.im/TheAlgorithms)! Community channels are a great way for you to ask questions and get help. Please join us!
```

## List of Algorithms

> See our [directory](DIRECTORY.md) for easier navigation and a better overview of the project.

Reviewer 2: you probably could already add an empty DIRECTORY.md file (or with some placeholder comment). Not really important since it will likely be fixed by a subsequent PR.

Author: I link this subsequent PR #5 is going to solve the problem. It proposes to automate the DIRECTORY.md management.
